### DISCUSSION TITLE
Tiempo ejecución tests

### DISCUSSION THREAD

**Discussion by sofialarrain** [IIC2133-PUC/2024-1#105] [Tarea 1] [answered]
Hola! Tengo una pregunta sobre el tiempo de ejecución. En el enunciado, se especifica que "tu programa deberá entregar el output correcto en menos de 10 segundos", mientras que en el bonus que subieron hoy día se dice que como referencia, el último test debe ejecutarse en menos de 10 segundos. Esto me confunde un poco. Si no deseo aplicar para el bonus y el tiempo de ejecución de todos mis tests es menor a 10 segundos, excepto el último, estoy en bien? O debería asegurarme de que el tiempo de ejecución sea menor a 10 segundos para todos los tests, incluyendo el último?

**Comment by sachondo7** [comment, marked as answer]
Holaa! Lo de 10 segundos en el bonus es solamente un parámetro que pusimos para que nadie rellenase el form porque si y los hagan quienes efectivamente creen cumplir con la complejidad del bonus. 
Respecto a lo otro, si intentaría optimizar tu código para que corra en menos de 10 segundos todos los tests ya que de lo contrario pueden haber algunos que no te den puntaje.
Saludos!
